AndroidApps/BundleTransport/app/src/main/java/net/discdd/bundletransport/screens/AppShareScreen.kt

```diff
@@ -80,7 +69,7 @@ fun AppShareScreen(
                             style = MaterialTheme.typography.titleMedium,
                             modifier = Modifier.padding(bottom = 16.dp)
                     )
-                } else {
+                } else if (clientApkSignature!! && mailApkSignature!!) {
```

Comment:
these can be null, i think you want clientApkSignature :? false && mailApkSignature ?: false